hw/ip/i2c/doc/_index.md

```diff
@@ -1,3 +1,223 @@
+
 ---
-title: "I2C Technical Specification" 
+title: "I2C HWIP Technical Specification"
 ---
+
+
+# Overview 
+
+This document specifies I2C hardware IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.](https://github.com/lowRISC/stwg-base/blob/master/doc/ComportabilitySpecification.md)
```

Comment:
Please replace this by

```
 conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
```

Reply:
Done, and hugo-tested (note link is slightly different than above)